src/server/models/audit.ts

```diff
@@ -18,6 +18,10 @@ interface ListItemEventData {
   updatedJsonData?: ListItemJsonData;
 }
 
+/**
+ * @deprecated
+ * TODO: deprecate, this is handled by the history field
+ */
```

Comment:
Do you think it's worth adding a separate ticket to get rid of this in the code?

Reply:
Yep. I'll add to backlog, thanks